risedev.yml

```diff
@@ -516,6 +528,15 @@ template:
     # Whether to disable recovery mode
     unsafe-disable-recovery: false
 
+    # Interval of GC stale metadata and SST
+    vacuum-interval-sec: 30
+
+    # The spin interval when collecting global GC watermark in hummock
+    collect-gc-watermark-spin-interval-sec: 5
+
+    # Threshold used by worker node to filter out new SSTs when scanning object store, during full SST GC.
+    min-sst-retention-time-sec: 604800
```

Comment:
Maybe we can leave a comment that default is 7 days

Reply:
fixed